Gopkg.toml

```diff
@@ -85,3 +85,7 @@
 [[constraint]]
   name = "github.com/asdine/storm"
   version = "2.1.2"
+
+[[constraint]]
+  branch = "master"
```

Comment:
better freeze to concrete commit from master - revision="<commit id>" to avoid surprises later

Reply:
Fixed.